src/assets/fonts.tsx

```diff
@@ -0,0 +1,11 @@
+import { injectGlobal } from 'styled-components'
+
+injectGlobal`
+    @font-face {
+        font-family: 'artsy-icons';
```

Comment:
@alloy  I opened a ticket for the issue @craigspaeth mentioned here https://github.com/artsy/reaction-force/issues/24 
So we can address this in a separate PR.